web_src/less/_base.less

```
@@ -332,6 +332,14 @@ a.commit-statuses-trigger {
   &:extend(.unselectable);
 }
 
+.issue-title {
+  code {
+    padding: 2px 4px;
+    border-radius: 6px;
+    background-color: var(--color-markup-code-block);
+  }
+}
+
```

No need to nest this.
```suggestion
.issue-title code {
  padding: 2px 4px;
  border-radius: 6px;
  background-color: var(--color-markup-code-block);
}
```